### ISSUE TITLE
Proposal: timer should start automatically when a file modification is detected in the repo

### ISSUE THREAD

**Issue by kalmard0** [timeglass/glass#46]
(see title)

**Comment by advdv** [comment]
Hey Kaliman, thanks for the feedback. I assume you mean when the timer is not currently paused right, cause that should work in the current version. Would you (or your IT guy) be Ok with a background processes running in the background continuously to monitor for these file changes?

**Comment by kalmard0** [comment]
Hi!
To be honest I've just started using glass 15 minutes ago, but my initial tests weren't successful. First I just added it to the project (glass init) then did some work and commit - nothing showed up in the commit message.
Next I checked out a new branch (and I saw that glass has started) did some work and commited it and again, nothing showed up, and 'glass status' said there was no timer running.

I'm on the newest release binary, Ubuntu 15.04 64.

Well, I'm my IT guy, and yes, I'm ok with a background process running. It's not like I read the source code of every single app I'm running anyway.

**Comment by advdv** [comment]
ok, lets see, could you run:  `git version`, `git log --show-notes=time-spent` and `glass status` and post the ouput here.

**Comment by kalmard0** [comment]
```
git version 2.1.4
```

```
commit c11b34a371456cdc12a837657ec9d18af25bf5b1
Author: Daniel Kalmar <kalmard@gmail.com>
Date:   Sun Jun 28 11:19:02 2015 +0200

    moved server project to pom

Notes (time-spent):
    total=25m0s
```

```
[8m0s]
```

It seems it's working fine then. I was under the impression that the commit messages would contain these times by default, but this is okay as well, as long as I can sum up the times later.

**Comment by advdv** [comment]
It should add the commit message by default, how does your `timeglass.json` looks like? how do you write commit messages, through the -m flag or using something else?

**Comment by kalmard0** [comment]
I don't have a timeglass.json file (the default settings seemed fine for testing it).
I write commit messages by calling 'git commit' which starts the default editor, mcedit.
I'm usually not working in the project root but sub-directories, maybe that is relevant.

**Comment by advdv** [comment]
Something unexpected seems to be happening when writing commit messages without -m flag. Thanks for reporting I'm opening a new issue for this.

**Comment by advdv** [comment]
@kalimaul This has been implemented in the latest release (0.6.0), please try it out and let me know how it goes.
